hw9/ArrayValueCalculator.java

```diff
@@ -0,0 +1,24 @@
+package org.hillel.hw9;
+
+import org.hillel.hw9.exceptions.ArrayDataException;
+import org.hillel.hw9.exceptions.ArraySizeException;
+
+public class ArrayValueCalculator {
+    public static int doCalc(Object[][] arr) throws ArraySizeException, ArrayDataException {
+        if (arr.length != 4 || arr[0].length != 4) {

```

Comment:
`arr.length != 4` <- this is correct. No questions.
But this -> `arr[0].length != 4` is not correct.
You've started in the correct way, but haven't finished the validation. You're validating only 1 row here, but it's required to validate them all because the matrix must be of size 4x4 according to the description.